recipes/imguizmo/all/conanfile.py

```diff
@@ -44,6 +43,12 @@ def source(self):
         extracted_dir = glob.glob("ImGuizmo-*/")[0]
         os.rename(extracted_dir, self._source_subfolder)
 
+    def requirements(self):
+        if self.version == "cci.20210223":
+            self.requires("imgui/1.82")
+        else:
+            self.requires("imgui/1.83")
```

Comment:
```suggestion
            self.requires("imgui/1.84.2")
```

Use the latest version if possible

Reply:
https://conan.io/center/imgui

Reply:
I think imguizmo/1.83 depends on imgui/1.83, because there are several breaking changes in imgui/1.84.
https://github.com/ocornut/imgui/releases/tag/v1.84

Reply:
Oh i see, it's aligned with imgui version. I read on imguizmo that 1.84 is under development.